vips/vip-332/bscmainnet.ts

```diff
@@ -0,0 +1,60 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const VUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+
+export const VANGUARD_VANTAGE_WALLET = "0xDeA0b46950dDc377E71800deba8F52456A7E42EE";
+export const TOKEN_REDEEMER = "0xC53ffda840B51068C64b2E052a5715043f634bcd";
+export const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const VANGUARD_VANTAGE_AMOUNT = parseUnits("542000", 18);
+export const VUSDC_AMOUNT = parseUnits("18750000", 8); // assuming 1 usdc = 0.024 vusdc redeem 450,000 usdc
+
+export const vip332 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-332 Transfer funds to fund the launch of the Vanguard Vantage Program",
+    description: `#### Summary
+
+If passed this VIP will perform the following actions:
+
+  - Redeem
+  - Transfer 542,000 USDC
+
+#### Details
+
+#### References
+  - Snapshot [Launch the Vanguard Vantage Program](https://snapshot.org/#/venus-xvs.eth/proposal/0x9728614f6c10a544569c3410356562dca8d1fb99ba9a582e752fa3a88569d01b)
+
+- [VIP simulation](https://github.com/VenusProtocol/vips/pull/312)`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [VUSDC, VUSDC_AMOUNT, TOKEN_REDEEMER],
+      },
+      {
+        target: TOKEN_REDEEMER,
+        signature: "redeemAndTransfer(address,address)",
+        params: [VUSDC, TREASURY],
+      },
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDC, VANGUARD_VANTAGE_AMOUNT, VANGUARD_VANTAGE_WALLET],
+      },
```

Comment:
We can use `redeemUnderlyingAndTransfer`:
* destination: `VANGUARD_VANTAGE_WALLET`
* receiver: Venus Treasury